lib/model/folder_sendrecv_test.go

```diff
@@ -415,6 +416,87 @@ func TestWeakHash(t *testing.T) {
 	}
 }
 
+func TestWeakHashWrite(t *testing.T) {
```

Comment:
Does this fail before the bug was fixed?

Reply:
Yes.